apps/web/src/app.tsx

```diff
@@ -110,7 +124,7 @@ function App() {
       </main>
 
       {showMembers && (
-        <div id="members">
+        <div id="members-sidebar">
```

Comment:
This is place I'm referring in previous comment regarding "sidebar-members" naming.